lib/spack/spack/build_systems/autotools.py

```diff
@@ -151,9 +151,32 @@ def configure(self, spec, prefix):
         options = ['--prefix={0}'.format(prefix)] + self.configure_args()
         inspect.getmodule(self).configure(*options)
 
+    def make_targets_test(self):
```

Comment:
I am not in favor of adding `make_targets_test`. The two common cases `check` and `test` are already in the base class. If you need to add targets that are specific to Blitz++ I would use a decorators in Blitz++ package to run something after the build phase when `--run-tests` is given.